allensdk/brain_observatory/behavior/mtrain.py

```diff
@@ -1,5 +1,6 @@
 from marshmallow import Schema, fields
 from datetime import datetime, date
+import numpy as np
```

Comment:
?

Reply:
This import was missing. See fix_change_time in line 47

Reply:
ah nice